### DISCUSSION TITLE
Office365 integration on data processing

### DISCUSSION THREAD

**Discussion by ecvetek** [utmstack/UTMStack#587] [Q&A] [answered]
Hi,

I see that every integration i enable appears on Data processing dashboard and shows number of events processed. This is not the fact with Office365 integration. Is this a bug or functionality by design? There is no visibility of events flowing in from Office365 connector?

Thanks,
EC

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @ecvetek, thanks for asking. There are a few integrations that aren't managed by our data processing server and won't be visible in that view:
- Office365
- Sophos Central
- AWS
- File Classification
- SOC AI

Those integration data don't need transformations, they enter our platform already transformed.

Best regards

**Reply by ecvetek** [reply]
Understood! Thank you @c3s4rfred.
